lib/ws-kernel-picker.js

```diff
@@ -91,7 +97,9 @@ export default class WSKernelPicker {
           this.onSession.bind(this)
         );
         sessionListing.previouslyFocusedElement = this.previouslyFocusedElement;
-        sessionListing.setLoading("Loading sessions...");
+        sessionListing.selectListView.update({
+          loadingMessage: "Loading sessions..."
+        });
```

Comment:
First, I'd set a breakpoint [here](https://github.com/BenRussert/hydrogen/blob/66fa4244b86d07c32c399e7cce24a430651c3bff/lib/ws-kernel-picker.js#L129) to make sure  the loading message is removed when you  think it should.

If, despite calling `update`, the loading message isn't cleared, then I'd try to call `sessionListing.selectListView.render()` explicitly.